packages/flutter/test/painting/decoration_test.dart

```diff
@@ -227,10 +227,10 @@ void main() {
     final Invocation call = canvas.invocations.singleWhere((Invocation call) => call.memberName == #drawImageNine);
     expect(call.isMethod, isTrue);
     expect(call.positionalArguments, hasLength(4));
-    expect(call.positionalArguments[0], const TypeMatcher<TestImage>());
+    expect(call.positionalArguments[0], const isInstanceOf<TestImage>());
```

Comment:
We should remove `const` in this file, right?

Reply:
That file doesn't use flutter_test, so no.